docs/UserGuide.md

```diff
@@ -214,6 +218,28 @@ Example |  Outcome
 --------|------------------
 Command: <br> `done 1` <br> Description: <br> Marks the item specified by `index 1` in the chore list as done. | done 1 <br> You have completed this chore: <br> [/] buy groceries (by: Monday 12pm) <br> <br> ===================================================
 
+### 3.5. Storage
+
+#### 3.5.1. Select Load Files
+Prompts the user with the option to either load their data from auto-save mode or the manual-save mode. If the user chooses the manual-save mode,  it will overwrite all the data stored in auto-save mode. However, any subsequent changes made to the program data will be saved through auto-save mode regardless of initial load options, to save through manual-save mode, user will have to use the save command [Section 3.5.3, "Save Current State"](#352-save-current-state-save).
+
+
+
+__Format:__ `addchore <task_description> /by <deadline>`  
+
+Example |  Outcome
+--------|------------------
+Command: <br> `1` <br> Description: <br> Data loaded from most recent auto-save mode files.  | Please enter '1' for auto-save and '2' for manual-save: <br> 1 <br> =================================================== <br> Okay auto-save chosen. <br> ===================================================
```

Comment:
Great job in adding the outputs that users are supposed to see when they key in the command!